src/settings/FixedDueDateSchedule/components/EditSections/components/SchedulesList/SchedulesList.test.js

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  within,
+  fireEvent,
+} from '@testing-library/react';
+import {
+  isArray,
+  uniqueId,
+} from 'lodash';
+
+import '../../../../../../../test/jest/__mock__';
+
+import SchedulesList from './SchedulesList';
+import ScheduleCard from '../ScheduleCard';
+
+jest.mock('../ScheduleCard', () => jest.fn(({
+  onRemoveSchedule,
+  scheduleIndex,
+}) => (
+  // eslint-disable-next-line
+  <div
+    data-testid="scheduleCard"
+    onClick={() => onRemoveSchedule(scheduleIndex)}
+  />
+)));
+jest.mock('lodash', () => {
+  const originalModule = jest.requireActual('lodash');
+
+  return {
+    __esModule: true,
+    ...originalModule,
+    uniqueId: jest.fn((prefix) => `${prefix}_uniqueId`),
+  };
+});
+
+describe('SchedulesList', () => {
+  const testTimeZone = 'UTC';
+  const testFieldsNames = [
+    'testSchedule1',
+    'testSchedule2',
+  ];
+  const testFieldsValue = [
+    {
+      key: 'testScheduleKey1',
+    },
+    {
+      key: 'testScheduleKey2',
+    },
+  ];
+  const testFields = {
+    push: jest.fn(),
+    remove: jest.fn(),
+    map: (callback) => testFieldsNames.map((name, index) => callback(name, index)),
```

Comment:
map: (callback) => testFieldsNames.map(callback) - should work, i think

Reply:
good point. done